backend/app/api/endpoints/admin/runtime_cleanup.py

```
@@ -0,0 +1,140 @@
+
+    return {
+        **result,
+        "reason": "delete_failed",
+        "error": error,
+    }
+
+
+@router.post("/runtime-cleanup/stale", response_model=Dict[str, Any])
+async def cleanup_stale_runtimes(
+    request: RuntimeCleanupRequest,
+    db: AsyncSession = Depends(get_async_db),
```

Ruff (0.15.13)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)